src/components/Compare/Filters.tsx

```diff
@@ -98,16 +99,10 @@ const Filters = (props: {
     }
   };
 
-  const valueMap: any = {
-    0: GeoScopeFilter.NEARBY,
-    50: GeoScopeFilter.STATE,
-    99: GeoScopeFilter.COUNTRY,
-  };
```

Comment:
Per my other comment, if we can move the sliderValueMap back into this file and keep the magic 0, 50, 99 values contained to this file, I think that'll be cleaner.

Hopefully external code can use the geoScope value directly.

Reply:
Ooo yes good call. Moving this map back into Filters.tsx